pcdsdevices/attenuator.py

```diff
@@ -27,6 +28,31 @@
 logger = logging.getLogger(__name__)
 MAX_FILTERS = 12
 
+class Gatt_Y_aperture(TwinCATStatePMPS):
+    """
+    Controls the Y axis component of the apertures inside AT1K0
+    Defines the state count as 4 (5.5mm, 8mm, 10mm, 13mm) to limit the number of
+    config PVs we connect to.
+    """
+
+    states_list = []
+    in_states = []
+    out_states = []
+    _in_if_not_out = True
+    config = UpCpt(state_count=4)
+
+class Gatt_X_aperture(TwinCATStatePMPS):
```

Comment:
You'll need two blank lines in between class definitions.
Did you set up pre-commit?

https://confluence.slac.stanford.edu/display/PCDS/How+to+add+a+device+to+pcdsdevices#Howtoaddadevicetopcdsdevices-Pre-commit

Reply:
I have now!